### ISSUE TITLE
Reply highlight is too low contrast

### ISSUE THREAD

**Issue by lampholder** [vector-im/element-web#19475]
### Steps to reproduce

1. Find a message that is a reply to another message
2. Click it to find the reply
3. See that the message the reply was in reply to gets a very pale grey highlight.

![image](https://user-images.githubusercontent.com/1922197/138462558-97dd5f0a-5c43-45c2-bfe6-f3d9c83edb6d.png)

This is particularly problematic when the message is on the same page of scrollback as the reply - if the timeline doesn't need to move to show you the message (and if you fail to notice the pale grey highlight) then it feels like nothing has happened when you click on the reply.

#### What did you expect?
Easy to see message highlighting.

#### What happened instead?
Difficult to see message highlighting.

### Operating system

Windwos

### Browser information

Firefox Nightly

### URL for webapp

develop.element.io

### Homeserver

lant.uk

### Will you send logs?

No

**Comment by t3chguy** [comment]
Modern layout uses `$event-selected-color: #f6f7f8;`
Bubble layout uses `$eventbubble-bg-hover: #FAFBFD;`

Needs design to pick a new colour

**Comment by amshakal** [comment]
Just checked, we can stick with #f6f7f8 for bubbles

**Comment by daniellekirkwood** [comment]
going to switch back the assignments now we have an answer :D

**Comment by t3chguy** [comment]
@amshakal doing that makes it blend with the colour of left-column bubbles `$eventbubble-self-bg`

![image](https://user-images.githubusercontent.com/2403652/146756398-7569add1-2e60-4241-89df-13867d967f16.png)

**Comment by amshakal** [comment]
Hello! Since we changed the bubble colour, we can now stick to #f6f7f8 for highlights so it's visible. It shouldn't blend in with the bubble or seem too low contrast.
